### PR TITLE
account resource provider

### PR DESCRIPTION
**Pull request by xgp**
Quickstart demonstrating overriding of Account Console resources 
Issue: https://github.com/keycloak/keycloak/issues/22318
PR: https://github.com/keycloak/keycloak/pull/22317

@garronej @ssilvert

### CODE REVIEW

File: extension/account-resource-provider/src/main/java/org/keycloak/forms/account/AccountProviderFactory.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.keycloak.forms.account;
+
+import org.keycloak.provider.ProviderFactory;
+
+/**
+ * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>
```
Comment (ssilvert) [inline extension/account-resource-provider/src/main/java/org/keycloak/forms/account/AccountProviderFactory.java:23]: You don't want Stian to take credit for your work, right?

**Review by ssilvert — COMMENTED** [review]
Minor tweak is that copyrights should be updated to year 2023.

Thanks for the contribution.  I'm going to try this out soon.

**Comment by mposolda** [comment]
@xgp There should not be PR sent to the branch `latest` . All PRs to quickstarts should go to branch `main` .

The content of branch `latest` is always overwritten from branch `main` during Keycloak release, so your changes to the branch `latest` would be lost if updates are done only in branch `latest` instead of updating branch `main` . We plan to update quickstarts contributon guide with mentioning this.

**Comment by garronej** [comment]
I hope this get merged at one point. It's great being able to override the Account Console resources.

**Comment by xgp** [comment]
> @xgp There should not be PR sent to the branch `latest` . All PRs to quickstarts should go to branch `main` .

Sorry for the mistake @mposolda I just assumed `latest` was supposed to be the target branch, as that was the default one that came up. I'll resubmit against `main`.

**Pull request closed without merging**